testing/pdal/APKBUILD

```diff
@@ -1,14 +1,14 @@
 # Contributor: Bradley J Chambers <brad.chambers@gmail.com>
 # Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
 pkgname=pdal
-pkgver=1.6.0
-pkgrel=1
+pkgver=1.7.0
+pkgrel=0
 pkgdesc="PDAL"
```

Comment:
pkgdesc could be improved. E.g. "PDAL is a C++ BSD library for translating and manipulating point cloud data."

Reply:
Done.